spring-cloud-services-spring-connector/src/main/java/io/pivotal/spring/cloud/service/config/PropertySourceMaskingConfiguration.java

```diff
@@ -0,0 +1,19 @@
+package io.pivotal.spring.cloud.service.config;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.cloud.config.client.ConfigServicePropertySourceLocator;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConditionalOnClass(ConfigServicePropertySourceLocator.class)
+public class PropertySourceMaskingConfiguration {
+
+	@Bean
+	public PropertySourceMaskingEnvironmentEndpoint environmentEndpoint() {
+		PropertySourceMaskingEnvironmentEndpoint environmentEndpoint = new PropertySourceMaskingEnvironmentEndpoint();
+		environmentEndpoint.setSourceNamePatterns("configService:vault:*");
```

Comment:
Does this include property sources resulting from composites containing vault backends?